job_spec/RIVER_WIDTH.yml

```diff
@@ -84,3 +90,6 @@ RIVER_WIDTH:
       timeout: 36000
       vcpu: 1
       memory: 31600
+      secrets:
+        - EARTHDATA_USERNAME
+        - EARTHDATA_PASSWORD
```

Comment:
:thinking: I'm not actually sure if `RiverWidthSAR` needs EDL credentials... will need to check.